limerick.txt

```diff
@@ -1,4 +1,4 @@
 Finish this limerick...
 
 There was a young lass from New York
-Who...
+Who etc. etc. blah blah blah
```

Comment:
Uh... This doesn't quite look like a limerick, @EleanorCallottWhitney 

Check out [this link](https://en.wikipedia.org/wiki/Limerick_(poetry)) to learn how to write one properly.

I'd love to see and review your work when you have a new version!

:star2: :+1: